content/en/docs/operations/virtualization/windows.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Running Windows VMs in Cozystack"
+linkTitle: "Windows VMs"
+description: "Running Windows VMs in Cozystack"
+weight: 15
+----------
+
+Cozystack can run Windows virtual machines.
+This guide explains the prerequisites and steps required to boot up a virtual machine running Windows OS.
+
+
+## Prerequisites
+
+-   Windows installation ISO image.
+-   Virtio drivers ISO image.
+-   KubeVirt client `virtctl` [installed in your local environment](https://kubevirt.io/user-guide/user_workloads/virtctl_client_tool/)
+    and configured for your tenant's namespace.
```

Comment:
@kvaps, I think we need a guide on setting up `virtctl` and accessing VMs, regardless of the OS they use.